payload/payload.cc

```diff
@@ -42,11 +42,13 @@ void createInitialGlobalState(unique_ptr<core::GlobalState> &gs, const realmain:
 }
 
 void retainGlobalState(unique_ptr<core::GlobalState> &gs, const realmain::options::Options &options,
-                       unique_ptr<KeyValueStore> &kvstore) {
+                       unique_ptr<OwnedKeyValueStore> kvstore, bool forceCommit) {
```

Comment:
arguments like `forceCommit` make it hard to reason about the code. 
Can we make the callsites that do this call `OwnedKeyValueStore::commitAndClose(move(kvstore))` dirrectly instead?

Also, why doesn't it emit a timer?

Reply:
Sure, I can do that.

> Also, why doesn't it emit a timer?

I can fix that.